packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx

```diff
@@ -62,47 +62,24 @@ const messages = {
   sendLabel: 'Send',
   transactionsLabel: 'View Transactions',
   audio: '$AUDIO',
-  manageWallets: 'Manage Wallets',
-  buyAudio: 'Buy $AUDIO',
-  buyAudioSubheader:
-    'Buy $Audio tokens to unlock VIP tiers, earn badges, and tip artists',
-  showAdvanced: 'Show More Options',
-  hideAdvanced: 'Hide More Options',
-  additionalPaymentMethods: 'Additional Payment Methods',
-  advancedOptions: 'Advanced Options',
-  stripeRegionNotSupported:
-    'Link by Stripe is not yet supported in your region',
-  coinbasePayRegionNotSupported:
-    'Coinbase Pay is not yet supported in your region',
-  stripeStateNotSupported: (state: string) =>
-    `Link by Stripe is not supported in the state of ${state}`,
-  coinbaseStateNotSupported: (state: string) =>
-    `Coinbase Pay is not supported in the state of ${state}`,
-  goldAudioToken: 'Gold $AUDIO token',
-  findArtists: 'Find Artists to Support on Trending'
+  externalWallets: 'External Wallets',
+  buyAudio: 'Buy $AUDIO Tokens',
+  buyAudioNotSupported: 'Buy $AUDIO is not yet supported in your region',
+  findArtists: 'Find Artists to Support on Trending',
+  onRampsPowered: 'Onramps powered by'
```

Comment:
nit on-ramps seems more commonly used but we've used onramps on our blog

Reply:
https://stripe.com/newsroom/news/fiat-to-crypto-onramp
🤷 🤷  we should be consisten tho, ill check in...